### ISSUE TITLE
"enter" key does not work on Ubuntu

### ISSUE THREAD

**Issue by bredmond** [asweigart/pyautogui#29]
Trying to use the 'enter' key on Ubuntu throws an error.  Using 'return' works just fine.  Using 'enter' on OS X also worked fine.
```
In [1]: import pyautogui
<class 'Xlib.protocol.request.QueryExtension'>

In [2]: pyautogui.press('enter')
X protocol error:
<class 'Xlib.error.BadValue'>: code = 2, resource_id = 0, sequence_number = 10, major_opcode = 132, minor_opcode = 2
X protocol error:
<class 'Xlib.error.BadValue'>: code = 2, resource_id = 0, sequence_number = 12, major_opcode = 132, minor_opcode = 2

In [3]: pyautogui.press('return')                                                                                                                                                   

In [4]: !lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.10
Release:        14.10
Codename:       utopic
```

**Comment by asweigart** [comment]
Fixed this in 0.9.29. Thanks!

**Comment by bredmond** [comment]
Thank you!
